src/config/environment-variables.ts

```diff
@@ -0,0 +1,6 @@
+export const envVars = {
+  microsEnv: process.env.MICROS_ENV,
+  microsServiceVersion: process.env.MICROS_SERVICE_VERSION,
+  nodeEnv: process.env.NODE_ENV,
+  sentryServerDSN: process.env.SENTRY_DSN,
+};
```

Comment:
We already have something for that in `env.ts`, we just need to add the typings to it.

Reply:
Do you mean that I should move this to env.ts?

Reply:
Yeah :)

It tries to read the .env and return the parsed env vars from that, but we could combine it to try to simplify and centralize all environment variables.

I'd also recommend we use the same name as the env vars we're passing in to remove confusion or typos: `SENTRY_DSN: process.env.SENTRY_DSN`

Reply:
What about in micros deployments? Won't it fail if I move envVars there? env.ts is at the root. Wouldn't we need to keep envVars inside src?

Reply:
To be clear, I mean `src/config/env.ts` :)

Reply:
oh haha. Didn't even realise that was there 🤦